utils/mil_tools/mil_ros_tools/geometry_helpers.py

```diff
@@ -1,6 +1,7 @@
 from __future__ import division
 import numpy as np
-from tf.transformations import quaternion_from_euler, euler_from_quaternion, random_quaternion
+from tf.transformations import quaternion_from_euler, euler_from_quaternion, random_quaternion, quaternion_multiply
+from tf.transformations import quaternion_matrix as tf_quaternion_matrix
```

Comment:
We don't really have these style of imports in our codebase. I think consistency is nice. if you just do `import tf.transformations as trans` then you could just do `trans.quaternion_multiply()`. That way it's clear that all of these share the same namespace.